### PR TITLE
Clustering sometimes gets NaN

### PR DESCRIPTION
**Pull request by kumagi**
Originally reported by Hiroshi Kuwajima's [post](https://groups.google.com/d/msg/jubatus/J8dDkXKj7JE/S8hE2HbM0v0J) 

There are devide-by-zero and log(0) calculation exists.
It appears values with limited distribution(like integer).
For first aid, it should use very small value instead of zero.

Some radical(or mathematical) solution will be needed further.

### CODE REVIEW

**Comment by gwtnb** [comment]
cpplint.

File: jubatus/core/driver/clustering_test.cpp [hunk 1]
```diff
@@ -54,29 +56,32 @@ class clustering_test
...
         new driver::clustering(
             shared_ptr<core::clustering::clustering>(
                 new core::clustering::clustering("dummy",
                                                  param.second,
-                                                 conf)),
+                                                 conf_)),
             make_fv_converter()));
   }
   void SetUp() {
+    feenableexcept(FE_DIVBYZERO | FE_INVALID | FE_OVERFLOW);
```
Comment (gwtnb) [inline jubatus/core/driver/clustering_test.cpp]: It seems that there is no feenableexcept on OS X.

Comment (kumagi) [inline jubatus/core/driver/clustering_test.cpp]: `feenableexcept` will be no longer needed, I'll delete it.

**Comment by kumagi** [comment]
I'll rebase this branch.

**Comment by gwtnb** [comment]
:+1:

**Pull request merged**